literature.bib

```diff
@@ -337,6 +337,38 @@ @mastersthesis{Guething23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2024 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{BRSK:JSS24,
+  author       = {Alexander Boll and
```

Comment:
please but authors into a single line for consistency

Reply:
also for titles, and also for the other new entry

Reply:
done